cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
```

Comment:
I was wondering why you push commented codes but got to know that they are generated

Reply:
This file is automatically generated by cypress framework